var/spack/repos/builtin/packages/iperf3/package.py

```diff
@@ -12,8 +12,12 @@ class Iperf3(AutotoolsPackage):
     maintained project."""
 
     homepage = "https://software.es.net/iperf/"
-    url = "https://github.com/esnet/iperf/archive/3.6.tar.gz"
+    url = "https://downloads.es.net/pub/iperf/iperf-3.16.tar.gz"
 
     license("BSD-3-Clause-LBNL")
 
+    version("3.16", sha256="cc740c6bbea104398cc3e466befc515a25896ec85e44a662d5f4a767b9cf713e")
```

Comment:
There is now also 3.17 as of Friday.

Reply:
Happy for Spack to default to 3.17. SHA256 is 
* `077ede831b11b733ecf8b273abd97f9630fd7448d3ec1eaa789f396d82c8c943`

as per:

* http://software.es.net/iperf/news.html#iperf-3-17-released